main.go

```diff
@@ -60,8 +61,9 @@ const (
 )
 
 var (
-	scheme   = runtime.NewScheme()
-	setupLog = ctrl.Log.WithName("setup")
+	scheme                = runtime.NewScheme()
+	setupLog              = ctrl.Log.WithName("setup")
+	ignoreNamespacesRegex []regexp.Regexp
```

Comment:
I think it will be better to make this `[]*regexp.Regexp`. I didn't realize `Compile()` returned a pointer.

Reply:
Also can you move this out of this global var section and on top of the for loop where it is populated?

Reply:
> Also can you move this out of this global var section and on top of the for loop where it is populated?

Since it is being referred outside of the function with the for loop where it is being populated, wouldn't it become inaccessible to the other functions if it was moved out of the global var section?

Reply:
Yes, which is fine. You can pass it around as an parameter.